roles/installer/templates/deployments/task.yaml.j2

```diff
@@ -142,6 +142,11 @@ spec:
         - image: '{{ _redis_image }}'
           imagePullPolicy: '{{ image_pull_policy }}'
           name: redis
+{% if redis_command %}
+          command: {{ redis_command }}
+{% endif %}
+{% if redis_args %}
+          args: {{ redis_args }}          
```

Comment:
This line already exists:
* https://github.com/ansible/awx-operator/pull/1427/files#diff-4d0fa82fa796130ee45f174f37b43ce5faf7092d12717ad484dbdf712ff24988R155

You'll probably want to set the default to 

```
["redis-server", "/etc/redis.conf"]
```

I think that should be the value for `redis_command`, not `redis_args`, but the only way to know for sure will be to deploy it and try it out once those changes are made.